app/src/data/2028.yml

```
@@ -23,10 +23,13 @@
   items:
   - name: Treashunt
     href: https://feove.github.io/IborenWebsite/
-    class: tile-medium text-dark 
-    background_image: https://raw.githubusercontent.com/feove/IborenWebsite/main/media/logo_v1.png
+    class: tile-medium text-dark
+    background_image: https://raw.githubusercontent.com/feove/IborenWebsite/main/media/white-background.gif
     style:
     - 'background-color: #FFFFFF'
+    - 'background-size: auto 100%'
+    - 'background-position-x: right'
+    - 'background-position-y: 5%'
     - 'background-size: 85%'
     - 'background-position-y: 15%'
```

Latest merge commit restored old CSS code that this PRs aims to replace.
```suggestion
    - 'background-size: auto 100%'
    - 'background-position-x: right'
    - 'background-position-y: 5%'
```